File: sum/sum.java
import java.io.*;

class sum{
    public static void main(String args[]){
        Scanner in = new Scanner(System.in);
        int a ,b , sum=0;
        System.out.println("Enter the numbers:\n");
        a = in.nextInt();
        b = in.nextInt();
        sum = a+b;
        System.out.println("sum="+sum);
    }
}

File: sum/sum.c
#include <stdio.h>
int main()
{
    int a, b, sum = 0;
    printf("Enter the numbers:\n");
    scanf("%d", &a);
    scanf("%d", &b);
    sum = a + b;
    printf("sum=%d", sum);
    return 0;
}